axlearn/open_api/metrics/tool_use_execution.py

```diff
@@ -232,6 +249,128 @@ def _compare_tool_calls(
     return True
 
 
+@dataclasses.dataclass
+class DetailedMatchResult:
+    """Represents the tool matches for different metrics."""
```

Comment:
It's probably worth clarifying in this docstring how users should interpret the fields, e.g. `lenient` vs `lenient_bow`.
```suggestion
    """Represents the tool matches for different metrics.
    
    Attributes:
        func_name_match: The predicted function name matches the target function name.
        ...
    """
```

Reply:
Done